src/App.tsx

```diff
@@ -53,10 +53,7 @@ const App: FC = () => {
     const setAxiosHeaders = async () => {
       // Set global config of axios requests
       Axios.defaults.baseURL = API_URL;
-      // Uncomment this when we use our backend only
```

Comment:
Siden noen endpoints skal være tilgjengelig for anonyme brukere kan vi ikke gjøre dette for alle likevel